inspector/src/lodCube.ts

```diff
@@ -2,30 +2,34 @@ import { Effect } from "babylonjs/Materials/effect";
 
 let name = 'lodCubePixelShader';
 let shader = `
+#ifdef GL_ES
+precision highp float;
+#endif
+
 varying vec2 vUV;
 uniform samplerCube textureSampler;
 uniform float lod;
 void main(void)
 {
-vec2 uv=vUV*2.0-1.0;
-#ifdef POSITIVEX
-gl_FragColor=textureCube(textureSampler,vec3(1.001,uv.y,uv.x),lod);
-#endif
-#ifdef NEGATIVEX
-gl_FragColor=textureCube(textureSampler,vec3(-1.001,uv.y,uv.x),lod);
-#endif
-#ifdef POSITIVEY
-gl_FragColor=textureCube(textureSampler,vec3(uv.y,1.001,uv.x),lod);
-#endif
-#ifdef NEGATIVEY
-gl_FragColor=textureCube(textureSampler,vec3(uv.y,-1.001,uv.x),lod);
-#endif
-#ifdef POSITIVEZ
-gl_FragColor=textureCube(textureSampler,vec3(uv,1.001),lod);
-#endif
-#ifdef NEGATIVEZ
-gl_FragColor=textureCube(textureSampler,vec3(uv,-1.001),lod);
-#endif
+    vec2 uv=vUV*2.0-1.0;
+    #ifdef POSITIVEX
+    gl_FragColor=textureCube(textureSampler,vec3(1.001,uv.y,uv.x),lod);
```

Comment:
why the 0.001 ? I am wondering where it impacts.

Reply:
I'm not sure, I based this code on the cube pass postprocess: https://github.com/BabylonJS/Babylon.js/blob/master/src/Shaders/passCube.fragment.fx. @deltakosh do you know why this is 1.001 rather than 1.0?